src/api/java/cvc/CVCApiException.java

```diff
@@ -0,0 +1,9 @@
+package cvc;
```

Comment:
The package name should probably be `cvc5` to be consistent with the C++ API.

Reply:
I am in favor of just `cvc`. Any way the package name is too short and we need a proper long name like 
`com...cvc5`. This can be done in future PR when the name is finalized.

Reply:
I am convinced now with cvc5 in case the user wants two jar files cvc5.jar and cvc6.jar in the future.
Changed the package name to cvc5.